server/storage/mvcc/hash.go

```diff
@@ -22,7 +22,7 @@ import (
 	"go.etcd.io/etcd/server/v3/storage/schema"
 )
 
-func unsafeHashByRev(tx backend.ReadTx, lower, upper revision, keep map[revision]struct{}) (uint32, error) {
+func unsafeHashByRev(tx backend.ReadTx, lower, upper int64, keep map[revision]struct{}) (uint32, error) {
```

Comment:
I wish revision was strongly typed: 

`type Revision int64` such that it's hard to miss-type between int and Revision.

Probably for another refactoring.

Reply:
That's the good point. Here I wanted to stop using `revision` type to ensure that only main revision is set. `revision` type allows setting sub revision could lead to hard to predict side effects.